mne/realtime/epochs.py

```diff
@@ -314,6 +314,7 @@ def _process_raw_buffer(self, raw_buffer):
         verbose = 'ERROR'
         sfreq = self.info['sfreq']
         n_samp = len(self._raw_times)
+        prev_samp = int(self._find_events_kwargs['min_samples'] - 1)
```

Comment:
> Hmm, while implementing this I found a thing I'm not quite sure if it's a bug or a feature. If the trigger channel is [5,5,5,0,0,0, ...] does this count as an event if min_samples is small enough? That is if the trigger channel is active at the beginning of the buffer is it an event?

I would say it is not an event because you don't see the onset. If there is an onset in the previous buffer, it is an event, but you should count it only once, not twice.

> This line https://github.com/mne-tools/mne-python/blob/master/mne/event.py#L322 calculates the diffs, but since the trigger is already high the only diff is when the signal goes low. Because of this there is no start for the trigger and the test gets stuck.

What do you mean by test getting stuck? It fails? If I understand correctly, you do not have the problem of two events at the buffer boundary because the second buffer does not have an onset. So, there is nothing to fix, right?

> Should I add one zero value at the start of the data and subtract 1 from the first_sample so that there is change when the actual data is checked or will this break something?

Yeah, I wouldn't touch this if possible ... unless it is also applicable to cases where you are not dealing with real-time data